.travis.yml

```diff
@@ -37,14 +38,16 @@ matrix:
             - ubuntu-toolchain-r-test
           packages:
             - g++-4.9
+            - libboost-all-dev
     - env: CXX=g++-5 CC=gcc-5
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-5
-    - env: CXX=clang++-4.0 CC=clang-4.0
+            - libboost-all-dev
+    - env: CXX=clang++-4.0 CC=clang-4.0 CXXFLAGS=-stdlib=libc++
```

Comment:
`CXXFLAGS` is used in the `RetrieveGoogleTest.cmake` file to build googletest with the proper linker library.  This was only an issue in **clang++-4.0**